db/schema.rb

```diff
@@ -12,6 +12,14 @@
 
 ActiveRecord::Schema.define(version: 2019_04_12_181326) do
 
+  create_table "owner_repairs", force: :cascade do |t|
+    t.integer "owner_id"
```

Comment:
Kwasi, this table should not be in schema.rb.  I think it is left over from something else you did.  I think you can get rid of it by doing a rails db:reset followed by a rails db:migrate